cibuildwheel/resources/pinned_docker_images.cfg

```diff
@@ -53,5 +53,5 @@ manylinux_2_24 = quay.io/pypa/manylinux_2_24_aarch64:2022-12-26-0d38463
 manylinux_2_28 = quay.io/pypa/manylinux_2_28_aarch64:2024.09.28-3
 
 [armv7l]
-musllinux_1_2 = quay.io/pypa/musllinux_1_2_armv7l:2024.09.22-4
+musllinux_1_2 = quay.io/pypa/musllinux_1_2_armv7l:2024.09.28-3
```

Comment:
I don't see how 2024.09.28 could have RC 3, when it was merged 4 hours ago in https://github.com/pypa/manylinux/pull/1686.